src/UserSpaceInstrumentation/TestLib.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef USER_SPACE_INSTRUMENTATION_TEST_LIB_H_
+#define USER_SPACE_INSTRUMENTATION_TEST_LIB_H_
+
+#include <string>
+
+namespace orbit_user_space_instrumentation {
+
+// Library functions for some trivial logging. This library is merely used in tests: A binary
+// produced by this code is checked in into the testdata folder. The test injects this binary into
```

Comment:
Do you actually need to put it into the testdata folder? The library ends up in the `lib` directory anyway and you can reference it. This is what we did with `LinuxTracingIntegrationTestPuppetSharedObject` and it works fine. It also makes sure that the .so and the code are actually always in sync.

Reply:
Thanks! Indeed this looks way better.

Adjusted comments and removed the binary in testdata .